frontend/web/components/pages/CreateOrganisationPage.tsx

```diff
@@ -0,0 +1,169 @@
+import Constants from 'common/constants'
+import PageTitle from 'components/PageTitle'
+import CondensedRow from 'components/CondensedRow'
+import InputGroup from 'components/base/forms/InputGroup'
+import Button from 'components/base/forms/Button'
+import AccountProvider from 'common/providers/AccountProvider'
+import API from 'project/api'
+import AppActions from 'common/dispatcher/app-actions'
+import Utils from 'common/utils/utils'
+// @ts-ignore
+import Project from 'common/project'
+import AccountStore from 'common/stores/account-store'
+import { Account } from 'common/types/responses'
+import CheckboxGroup from 'components/base/forms/CheckboxGroup'
+
+const CreateOrganisationPage: React.FC = () => {
+  const [name, setName] = useState<string>('')
+  const inputRef = useRef<HTMLInputElement | null>(null)
+  const focusTimeout = useRef<ReturnType<typeof setTimeout> | null>(null)
+  const history = useHistory()
+  const [hosting, setHosting] = useState(['public_saas'])
+
+  useEffect(() => {
+    API.trackPage(Constants.pages.CREATE_ORGANISATION)
+    focusTimeout.current = setTimeout(() => {
+      inputRef.current?.focus()
+      focusTimeout.current = null
+    }, 500)
+
+    return () => {
+      if (focusTimeout.current) {
+        clearTimeout(focusTimeout.current)
+      }
+    }
+  }, [])
+
+  const onSave = (id: string | number) => {
+    AppActions.selectOrganisation(id)
+    API.setCookie('organisation', `${id}`)
+
+    if (Utils.isSaas()) {
+      API.trackTraits({
+        hosting_preferences: hosting,
+      })
+    }
+
+    if (Utils.getFlagsmithHasFeature('welcome_page')) {
+      history.push('/getting-started')
+    } else {
+      history.push(Utils.getOrganisationHomePage(id))
+    }
+  }
+
+  if (
+    Project.superUserCreateOnly &&
+    // @ts-ignore
+    !(AccountStore.model as Account)?.is_superuser
+  ) {
+    return (
+      <div className='text-center alert'>
+        Your Flagsmith instance is setup to only allow super users to create an
+        organisation, please contact your administrator.
+      </div>
+    )
+  }
+
+  if (Utils.getFlagsmithHasFeature('disable_create_org')) {
+    return (
+      <div id='create-org-page' className='container app-container'>
+        This Flagsmith instance is configured to prevent additional
+        organisations from being created. Please contact an administrator. If
+        you think you are seeing this page by mistake, please check you are
+        invited to the correct organisation.
+      </div>
+    )
+  }
+
+  return (
+    <div id='create-org-page' className='container app-container'>
+      <PageTitle title='Create your organisation'>
+        Organisations allow you to manage multiple projects within a team.
+      </PageTitle>
+      <AccountProvider onSave={onSave}>
+        {(
+          { isSaving }: { isSaving: boolean },
+          { createOrganisation }: { createOrganisation: (name: any) => void },
+        ) => (
```

Comment:
It's a bit extra work and not so much in the scope but now that it's a FC we could get rid of the provider and use a hook ?

Reply:
I'll listen to the account store in hook form but I think replacing create org is a much bigger task suited for its own PR.